addons/beehave/nodes/beehave_tree.gd

```diff
@@ -124,6 +124,8 @@ func _physics_process(delta: float) -> void:
 
 
 func tick() -> int:
+	if actor == null:
```

Comment:
there was a scenario where you can set a `null` actor on a behaviour tree. In this case, the tree should not break the game but show a configuration warning.